guillotina/api/service.py

```diff
@@ -37,7 +37,9 @@ def __setattr__(self, name, value):
             return super().__setattr__(name, value)
 
         if name == self.__field_name:
-            getattr(self.__context, name)[self.__key] = value
+            val = getattr(self.__context, name)
+            setattr(self.__context, name, val)
```

Comment:
Why do we need to set again the dict?

Reply:
Because of default field values.

This is used in the multi file behavior.

Reply:
basically, this was causing values to not be saved